src/votekit/metrics.py

```diff
@@ -0,0 +1,146 @@
+from votekit.cvr_loaders import rank_column_csv
```

Comment:
what is the rest of the code in this file, is it just scratch work/testing materials?